Standards/scs-0412-v1-metering-json.md

```diff
@@ -0,0 +1,81 @@
+---
+title: JSON format for metering data
```

Comment:
I wondered wether the title should carry a reference to the IaaS layer as well, since this document specifically deals with the format for that layer, so there is the chance there will be one for other parts as well.

Reply:
Fixed.